### PR TITLE
feat(cce): add new resource cluster log config

### PR DESCRIPTION
**Pull request by Jason-Zhang9309**
**What this PR does / why we need it**:

**Which issue this PR fixes**:
*(optional, in `fixes #<issue number>(, fixes #<issue_number>, ...)` format, will close that issue when PR gets merged)*
fixes #xxx

**Special notes for your reviewer**:

**Release note**:

```release-note

```

## PR Checklist

* [x] Tests added/passed.

```
make testacc TEST='./huaweicloud/services/acceptance/cce' TESTARGS='-run TestAccClusterLogConfig_basic'
==> Checking that code complies with gofmt requirements...
TF_ACC=1 go test ./huaweicloud/services/acceptance/cce -v -run TestAccClusterLogConfig_basic -timeout 360m -parallel 4
=== RUN   TestAccClusterLogConfig_basic
=== PAUSE TestAccClusterLogConfig_basic
=== CONT  TestAccClusterLogConfig_basic
--- PASS: TestAccClusterLogConfig_basic (570.99s)
PASS
ok      github.com/huaweicloud/terraform-provider-huaweicloud/huaweicloud/services/acceptance/cce       571.639s
```

* [x] Documentation updated.
* [x] Schema updated.
* [x] CheckDeleted.

  - **a. During query operation (Read Context)**
    aa. Resource not found
image

    
  - **b. During delete/disassociate/unbind operation (Delete Context)**
    ba. Resource not found
image

### CODE REVIEW

File: huaweicloud/services/cce/resource_huaweicloud_cce_cluster_log_config.go [hunk 1]
```diff
@@ -0,0 +1,260 @@
...
+
+func buildLogConfigsCreateBodyParams(d *schema.ResourceData) []map[string]interface{} {
+	logConfigsRaw := d.Get("log_configs").(*schema.Set).List()
+	if len(logConfigsRaw) == 0 || logConfigsRaw[0] == nil {
+		return nil
+	}
+
+	bodyParams := make([]map[string]interface{}, len(logConfigsRaw))
+	for i, v := range logConfigsRaw {
+		logConfig := v.(map[string]interface{})
```
Comment (houpeng80) [inline huaweicloud/services/cce/resource_huaweicloud_cce_cluster_log_config.go]: if name and enable are both empty, then a panic will be occured

Comment (Jason-Zhang9309) [reply huaweicloud/services/cce/resource_huaweicloud_cce_cluster_log_config.go]: fixed

**Comment by Jason-Zhang9309** [comment]
```
make testacc TEST='./huaweicloud/services/acceptance/cce' TESTARGS='-run TestAccClusterLogConfig_basic'
==> Checking that code complies with gofmt requirements...
TF_ACC=1 go test ./huaweicloud/services/acceptance/cce -v -run TestAccClusterLogConfig_basic -timeout 360m -parallel 4
=== RUN   TestAccClusterLogConfig_basic
=== PAUSE TestAccClusterLogConfig_basic
=== CONT  TestAccClusterLogConfig_basic
--- PASS: TestAccClusterLogConfig_basic (599.62s)
PASS
ok      github.com/huaweicloud/terraform-provider-huaweicloud/huaweicloud/services/acceptance/cce       599.661s
```

**Pull request merged**